src/main/kotlin/io/snyk/plugin/ui/SnykBalloonNotifications.kt

```diff
@@ -23,22 +21,31 @@ class SnykBalloonNotifications {
         private const val groupAutoHide = "SnykAutoHide"
         private val GROUP = NotificationGroup(groupNeedAction, NotificationDisplayType.STICKY_BALLOON)
 
-        fun showError(message: String, project: Project, action: AnAction? = null) {
+        fun showError(message: String, project: Project, action: AnAction? = null)  =
+            showNotification(message, project, NotificationType.ERROR, action)
+
+        fun showInfo(message: String, project: Project, action: AnAction? = null) =
+            showNotification(message, project, NotificationType.INFORMATION, action)
+
+        fun showWarn(message: String, project: Project, action: AnAction? = null) =
+            showNotification(message, project, NotificationType.WARNING, action)
+
+        private fun showNotification(message: String, project: Project, type: NotificationType, action: AnAction?) {
             val notification = if (action == null) {
-                Notification(groupAutoHide, title, message, NotificationType.ERROR)
+                Notification(groupAutoHide, title, message, type)
             } else {
-                GROUP.createNotification(title, message, NotificationType.ERROR).addAction(action)
+                GROUP.createNotification(title, message, type).addAction(action)
             }
             notification.notify(project)
         }
 
-        fun showInfo(message: String, project: Project) {
-            Notification(groupAutoHide, title, message, NotificationType.INFORMATION).notify(project)
-        }
-
-        fun showWarn(message: String, project: Project) {
-            Notification(groupAutoHide, title, message, NotificationType.WARNING).notify(project)
-        }
+        fun showSastForOrgEnablement(project: Project) = showInfo(
+            "Snyk Code is disabled by your organisation's configuration. To enable navigate to ",
+            project,
+            NotificationAction.createSimpleExpiring("Snyk > Settings > Snyk Code"){
+                BrowserUtil.browse("https://app.snyk.io/manage/snyk-code")
```

Comment:
@ArtsiomCh , should we not use endpoint here? It could be that dev environment are configured (e.g. app.dev.snyk.io)